frontend/src/app/testQueue/TestCard/TestCard.test.tsx

```diff
@@ -41,6 +41,13 @@ jest.mock("../../TelemetryService", () => ({
   getAppInsights: jest.fn(),
 }));
 
+const mockDiseaseEnabledFlag = (diseaseName: string) =>
+  jest
+    .spyOn(flaggedMock, "useFeature")
+    .mockImplementation((flagName: string) => {
+      return flagName === `${diseaseName.toLowerCase()}Enabled`;
+    });
+
```

Comment:
building this mock out to be able to specify which flag to return true / false to so that we can test the HIV and Syphilis mocks out separately.

Reply:
Great addition here!